src/blocks/_pro/dynamic-text/index.php

```diff
@@ -0,0 +1,110 @@
+<?php
+/**
+ * Registers the `vk-blocks/dynamic-text` block.
+ *
+ * @package vk-blocks
+ */
+
+use VektorInc\VK_Helpers\VkHelpers;
+
+/**
+ * Dynamic text render callback
+ *
+ * @param array $attributes Block attributes.
+ * @return string
+ */
+function vk_blocks_dynamic_text_render_callback( $attributes ) {
+	$options = array(
+		'displayElement' => $attributes['displayElement'],
+		'tagName'        => $attributes['tagName'],
+	);
+
+	$post = get_post();
+	if ( 'ancestor-page' === $options['displayElement'] && ! ( $post->post_parent ) && 'page' !== $post->post_type ) {
```

Comment:
@doshimaf ありがとうございます。
たぶんここは === の間違いかと（・ｗ・

```suggestion
	if ( 'ancestor-page' === $options['displayElement'] && ! ( $post->post_parent ) && 'page' === $post->post_type ) {
```